src/Examples/UnitModels/Operations/skeleton_unit_testing.ipynb

```diff
@@ -0,0 +1,420 @@
+    "\n",
+    "# dictionaries relating state properties to custom variables\n",
+    "inlet_dict = {\"flow_mol_comp\": m.fs.skeleton.flow_comp_in,\n",
+    "              \"temperature\": m.fs.skeleton.temperature_in,\n",
+    "              \"pressure\": m.fs.skeleton.pressure_in}\n",
+    "outlet_dict = {\"flow_mol_comp\": m.fs.skeleton.flow_comp_out,\n",
+    "               \"temperature\": m.fs.skeleton.temperature_out,\n",
+    "               \"pressure\": m.fs.skeleton.pressure_out,}\n",
+    "\n",
+    "m.fs.skeleton.add_ports(name=\"inlet\", member_dict=inlet_dict)\n",
+    "m.fs.skeleton.add_ports(name=\"outlet\", member_dict=outlet_dict)\n",
+    "\n",
+    "# custom initialization scheme\n",
+    "m.fs.skeleton.inlet.flow_mol_comp[0, \"c1\"].fix(2*pyunits.mol/pyunits.s)\n",
+    "m.fs.skeleton.inlet.flow_mol_comp[0, \"c2\"].fix(2*pyunits.mol/pyunits.s)\n",
+    "m.fs.skeleton.inlet.temperature.fix(325*pyunits.K)\n",
+    "m.fs.skeleton.inlet.pressure.fix(200*pyunits.kPa)\n",
+    "\n",
+    "assert degrees_of_freedom(m) == 0\n",
+    "\n",
+    "def my_initialize(unit, **kwargs):\n",
+    "    # Callback for user provided initialization sequence\n",
+    "    unit.eq_temperature.deactivate()\n",
+    "    unit.eq_pressure.deactivate()\n",
+    "    unit.outlet.temperature.fix(325*pyunits.K)\n",
+    "    unit.outlet.pressure.fix(200*pyunits.kPa)\n",
+    "\n",
+    "    if degrees_of_freedom == 0:\n",
+    "        solver.solve(unit)\n",
+    "\n",
+    "    unit.eq_temperature.activate()\n",
+    "    unit.eq_pressure.activate()\n",
+    "    unit.outlet.temperature.unfix()\n",
+    "    unit.outlet.pressure.unfix()\n",
+    "\n",
+    "m.fs.skeleton.config.initializer = my_initialize\n",
+    "m.fs.skeleton.initialize()\n",
+    "\n",
+    "solver = SolverFactory('ipopt')\n",
+    "results = solver.solve(m, tee = True)\n",
+    "\n",
+    "assert check_optimal_termination(results)\n",
+    "assert_units_consistent(m)\n",
+    "\n",
+    "assert value(m.fs.skeleton.outlet.flow_mol_comp[0, \"c1\"]) == pytest.approx(2, abs=1e-3)\n",
+    "assert value(m.fs.skeleton.outlet.flow_mol_comp[0, \"c2\"]) == pytest.approx(2, abs=1e-3)\n",
+    "assert value(m.fs.skeleton.outlet.temperature[0]) == pytest.approx(335, abs=1e-3)\n",
+    "assert value(m.fs.skeleton.outlet.pressure[0]) == pytest.approx(190, abs=1e-3)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ],
+ "metadata": {
+  "celltoolbar": "Tags",
+  "kernelspec": {
+   "display_name": "Python 3 (ipykernel)",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.12"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```

Comment:
I think there needs to be some sort of summary or conclusion at the end to remind users of what they just read.